feat: improvements on WAI and related standards/recommendations (#34146)
mdn/content: files/en-us/glossary/wai/index.md

{{GlossarySidebar}}

> WAI or Web {{glossary("Accessibility")}} Initiative is an effort by the {{glossary("W3C")}} to develop standards and guidelines to help everyone build a web by improving accessibility for people with various challenges.

Reviewer: ```suggestion
The **Web Accessibility Initiative (WAI)** is an effort by the {{glossary("W3C")}} to develop standards and guidelines to help everyone building the web ensure and improve {{glossary("accessibility")}} for people with various challenges.
```
* Move the "accessiblity" macro link out of the names of organizations, making the next occurrence of the link the 
* The first mention of the org should be the full name, followed by the acronym; then the acronym can be used freely.

> WAI develops guidelines and other technical reports through the same process as other parts of the W3C. Accessibility guidelines developed include web content ({{glossary("WCAG")}}), authoring tools ({{glossary("ATAG")}}), and user agents ({{glossary("UAAG")}}). WAI also develops the Accessible Rich Internet Applications ({{Glossary("ARIA")}}) suite of technologies to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.

Reviewer: ```suggestion
WAI develops guidelines and other technical reports through the same process as other parts of the W3C. WAI develops accessibility guidelines for web content ({{glossary("WCAG")}}), authoring tools ({{glossary("ATAG")}}), and user agents ({{glossary("UAAG")}}). WAI also develops the Accessible Rich Internet Applications ({{Glossary("ARIA")}}) suite of technologies to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.
```
